arm-sql/2015-05-01-preview/swagger/failoverGroups.json

```diff
@@ -524,50 +605,41 @@
     "SubscriptionIdParameter": {
       "name": "subscriptionId",
       "in": "path",
-      "description": "The subscription ID that identifies an Azure subscription.",
+      "description": "The Azure subscription ID.",
       "required": true,
       "type": "string"
     },
     "ApiVersionParameter": {
       "name": "api-version",
       "in": "query",
-      "description": "The API version to use for the request.",
+      "description": "Client Api Version.",
       "required": true,
-      "x-ms-parameter-location": "client",
-      "type": "string"
+      "type": "string",
+      "x-ms-parameter-location": "client"
     },
     "ResourceGroupParameter": {
       "name": "resourceGroupName",
       "in": "path",
-      "description": "The name of the resource group that contains the resource. You can obtain this value from the Azure Resource Manager API or the portal.",
+      "description": "The name of the resource group that contains the resource.",
       "required": true,
-      "x-ms-parameter-location": "method",
-      "type": "string"
+      "type": "string",
+      "x-ms-parameter-location": "method"
     },
-    "ServerParameter": {
+    "ServerNameParameter": {
```

Comment:
We will soon be merging sql swagger files into a single file in order to move away from composite swagger, so this parameter will be used by other operations in that combined file. Do you mind if we keep this standard parameter for now?